test/test_other.py

```diff
@@ -10269,6 +10269,15 @@ def test_closure_externs(self):
                         '--pre-js', test_file('test_closure_externs_pre_js.js')] +
                        args)
 
+  # Tests that ports can add closure args by using settings.CLOSURE_ARGS
+  @crossplatform
+  def test_closure_args_from_port(self):
+    port = test_file('test_closure_args_port.py')
+    self.run_process([EMCC, test_file('hello_world.c'),
+                      f'--use-port={port}',
```

Comment:
Does `,...,'--port', test_file('test_closure_args_port.py') ..` work here

Reply:
Absolutely. The only purpose of this port is to set the closure args (and obviously test the feature in the process). If this line is commented out the test fails because the closure args are not set on the command line.

Reply:
What I mean is can you avoid the `port` variable here and do:

```
self.run_process([EMCC, test_file('hello_world.c'), '--use-port', test_file('test_closure_args_port.py'), ...
```

Reply:
i.e. no need for the f-string

Reply:
Ok I fixed it